sr_robot_launch/scripts/timed_roslaunch.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash          
+#
```

Comment:
This timed_roslaunch.sh is already in sr_interface in https://github.com/shadow-robot/sr_interface/tree/kinetic-devel/sr_multi_moveit/sr_multi_moveit_config/scripts so it doesn't need to be included again